nixos/modules/services/web-apps/lotide.nix

```
@@ -0,0 +1,104 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.lotide;
+in
+{
+  options.services.lotide = {
+    enable = mkEnableOption "the lotide service";
+
+    package = mkOption {
+      default = pkgs.lotide;
+      defaultText = "pkgs.lotide";
+      type = types.package;
+      description = "lotide package to use.";
+    };
+
+    databaseUrl = mkOption {
+      default = null;
+      type = types.string;
+      description = "Credentials for database connection.";
+      example = "postgresql://user:passwort@somehost:5432/databasename";
+    };
+
+    hostUrlActivityPub = mkOption {
+      default = null;
+      type = types.string;
+      description = "If using the recommended proxy setup, set this to your root address";
+      example = "https://example.com";
+    };
+
+    hostUrlAPI = mkOption {
+      default = null;
+      type = types.string;
+      description = "URL for API";
+
+      example = "https://example.com/api";
+    };
+
+    apubProxyRewrites= mkOption {
+      default = false;
+      type = types.bool;
+      description = "Set to `true` to make signatures work with the proxy setup.";
```

```suggestion
      description = "Set to <literal>true</literal> to make signatures work with the proxy setup.";
```

At least I think... I'm not a docbook person :man_shrugging: